appointment-frontend/src/components/feedback/Feedback.vue

```diff
@@ -357,13 +357,13 @@ export default class Feedback extends Vue {
     this.feedbackMessage = 'Feedback Message: ' + this.feedbackMessage + '\n'
     this.feedbackMessage = nonStepperLocation ? this.feedbackMessage + 'Step: ' + nonStepperLocation : this.feedbackMessage + 'Step: ' + this.bookingStepInfo[appointmentStep]
     this.feedbackMessage = appointmentLocation ? this.feedbackMessage + '\n' + 'Location: ' + appointmentLocation : this.feedbackMessage
-    this.feedbackRequest.variables.citizen_comments.value = this.feedbackMessage + '\n' + CommonUtils.getUserAgent()
+    this.feedbackRequest.variables.citizenComments.value = this.feedbackMessage + '\n' + CommonUtils.getUserAgent()
     this.feedbackRequest.variables.response.value = this.responseRequired ? 'true' : 'false'
-    this.feedbackRequest.variables.citizen_name.value = this.citizenName === '' ? 'None' : this.citizenName
-    this.feedbackRequest.variables.citizen_contact.value = this.phone === '' ? 'None' : this.phone
-    this.feedbackRequest.variables.citizen_email.value = this.email === '' ? 'None' : this.email.trim()
-    this.feedbackRequest.variables.service_date.value = this.getCurrentDateinFormat()
-    this.feedbackRequest.variables.submit_date_time.value = this.getCurrentDateinFormat()
+    this.feedbackRequest.variables.citizenName.value = this.citizenName === '' ? 'None' : this.citizenName
+    this.feedbackRequest.variables.citizenContact.value = this.phone === '' ? 'None' : this.phone
+    this.feedbackRequest.variables.citizenEmail.value = this.email === '' ? 'None' : this.email.trim()
+    this.feedbackRequest.variables.serviceDate.value = this.getCurrentDateinFormat()
+    this.feedbackRequest.variables.submitDateTime.value = this.getCurrentDateinFormat()
     const resp = await this.submitFeedback(this.feedbackRequest)
     if (resp.status) {
       if (resp.status === 200 && resp.data.response_code === 200) {
```

Comment:
@ChrisGabel23 - In dev when you send feedback, it always says "Feedback submission failed." although it is successfully in ServiceFlow, and the response is 200/200. I'm guessing that the problem is that this "response_code" is being changed to "responseCode" by humps.

Reply:
Looks good!

![image](https://user-images.githubusercontent.com/35532993/185489019-544e7e14-79ba-497c-a9bb-32ee09302339.png)